### PR TITLE
fix: order by custom attribute throws error in list view

### PR DESCRIPTION
**Pull request by nityanandagohain**
Fixes https://github.com/SigNoz/signoz/issues/6367

----

> [!IMPORTANT]
> Remove backticks from column names in SQL queries for `PanelTypeList` in `query_builder.go` and update tests accordingly.
> 
>   - **Behavior**:
>     - Removed backticks around column names in `orderBy()` in `query_builder.go` and `query_builder_test.go` for `PanelTypeList`.
>     - Updated expected query strings in `TestPrepareLogsQuery` and `Test_orderByAttributeKeyTags` in `query_builder_test.go` to reflect this change.
>   - **Tests**:
>     - Modified `Test_orderByAttributeKeyTags` to expect column names without backticks for `PanelTypeList`.
>     - Updated `TestPrepareLogsQuery` to expect queries without backticks around `timestamp` in order by clauses.
> 
> <sup>This description was created by </sup>[Ellipsis](https://www.ellipsis.dev?ref=SigNoz%2Fsignoz&utm_source=github&utm_medium=referral)<sup> for bc7d8260466c3fb81435c47e49af84e4dd8f1950. It will automatically update as commits are pushed.</sup>

### CODE REVIEW

File: pkg/query-service/app/logs/v4/query_builder.go
Comment (nityanandagohain) [reply pkg/query-service/app/logs/v4/query_builder.go:260]: So here in v4, `getClickhouseKey` returns the keys with "`"

Comment (srikanthccv) [reply pkg/query-service/app/logs/v4/query_builder.go:260]: What happens in v3? Do we manually add "`" after each call to "getClickhouse*" functions?

Comment (nityanandagohain) [reply pkg/query-service/app/logs/v4/query_builder.go:260]: This iss was present in v3 as well, have fixed it

**Comment by srikanthccv** [comment]
I am finding it hard to make sense of the **\`\`** around the code. Sometimes, they come from functions like `getClickhouseKey`, but sometimes, they appear standalone https://github.com/SigNoz/signoz/blob/fdc54a62a90348143629b3376da0a9bd60a6c3ad/pkg/query-service/app/logs/v4/query_builder.go#L254. And some places they are trimmed away because of _exist formatting https://github.com/SigNoz/signoz/blob/fdc54a62a90348143629b3376da0a9bd60a6c3ad/pkg/query-service/app/logs/v4/query_builder.go#L108C47-L108C51

**Comment by srikanthccv** [comment]
Please resolve the tests I will approve.

**Comment by nityanandagohain** [comment]
Agree, we can go ahead with this for now. Let me create another issue to see if we can simplify it.

**Review by srikanthccv — APPROVED** [review]

**Comment by srikanthccv** [comment]
You could have renamed the commit message before pressing the auto squash merge button. I wouldn't have cared if it was internal repo. The commit message here goes into the release notes.

**Comment by nityanandagohain** [comment]
Ahh, my bad, I skipped to notice it while enabling auto squash.

**Pull request merged**
